### PR TITLE
Citations (#44853)

### PR DESCRIPTION
**Pull request by Cenrus**
https://github.com/tgstation/tgstation/pull/44853

Citations from /tg/station, officers can fine people over petty crimes instead of arresting them.
fixed version of #1065

## Why it's good for the game

More things for security to do during slow extended shifts.

:cl: Atlanta Ned
add: Security can now issue citations! Spot someone committing a minor crime? Fine them for it!
add: Got a citation? You can view and pay it at any security warrant consoles, located near the brig, your local law office, or the courtroom!
/:cl:

### CODE REVIEW

**Comment by AyyRobotics** [comment]
I like that this implies security can take down someone for their debt

File: code/game/machinery/computer/warrant.dm [hunk 1]
```diff
@@ -0,0 +1,149 @@
+/obj/machinery/computer/warrant//TODO:SANITY
```
Comment (qwertyquerty) [inline code/game/machinery/computer/warrant.dm:1]: `//TODO:SANITY`

:vibecheck:

File: code/game/machinery/computer/warrant.dm [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+	desc = "Used to view crewmember security records"
+	icon_screen = "security"
+	icon_keyboard = "security_key"
+	circuit = /obj/item/circuitboard/computer/warrant
+	light_color = LIGHT_COLOR_RED
+	var/authenticated = null
+	var/screen = null
+	var/datum/data/record/current = null
+
+/obj/machinery/computer/warrant/ui_interact(mob/user)
```
Comment (qwertyquerty) [inline code/game/machinery/computer/warrant.dm:12]: This should really be done via tgui

Comment (ike709) [reply code/game/machinery/computer/warrant.dm:12]: I'm not making him convert it when the original PR didn't have it.

File: code/game/machinery/computer/warrant.dm (hunk 1 above)
Comment (ike709) [reply code/game/machinery/computer/warrant.dm:1]: Also in the original PR.

**Comment by ike709** [comment]
Seems fine. I'll take care of the wiki changes since I believe it's a protected page.

**Comment by ike709** [comment]
Turns out the wiki already had the relevant changes.

**Pull request merged**